start_time_intent_handler.py

```diff
@@ -69,7 +69,7 @@ def HandleIntent(self, post_params):
       canonical_prayer = util.StringToDailyPrayer(desired_prayer)
       iqama_time = GetIqamaTime(desired_prayer,masjid)
       print 'iqama_time[', desired_prayer, "] = ", iqama_time
-      return self._MakeSpeechResponse(canonical_prayer, desired_prayer, iqama_time, masjid)
+      return self._MakeSpeechResponse(canonical_prayer, desired_prayer, iqama_time, [masjid,"Masjid"])
```

Comment:
what about making this a tuple of enum and string?

ex: (LocationType::MOSQUE, masjid_name) or (LocationType::CITY, city_name)